lib/matplotlib/tests/baseline_images/test_backend_ps/useafm.eps

```diff
@@ -0,0 +1,66 @@
+%!PS-Adobe-3.0 EPSF-3.0
+%%Title: /home/antony/src/extern/matplotlib/result_images/test_backend_ps/useafm.eps
```

Comment:
This seems a bit leaky; I wonder if we should trim the title to just the filename?

Reply:
As a general matter (i.e. not just for tests), I would say yes.